op-batcher/batcher/sync_actions.go

```diff
@@ -16,17 +16,18 @@ type channelStatuser interface {
 	MaxInclusionBlock() uint64
 }
 
+type inclusiveBlockRange struct{ start, end uint64 }
 type syncActions struct {
 	clearState      *eth.BlockID
 	blocksToPrune   int
 	channelsToPrune int
-	blocksToLoad    [2]uint64 // the range [start,end] that should be loaded into the local state.
+	blocksToLoad    *inclusiveBlockRange // the blocks that should be loaded into the local state.
 	// NOTE this range is inclusive on both ends, which is a change to previous behaviour.
 }
 
 func (s syncActions) String() string {
 	return fmt.Sprintf(
-		"SyncActions{blocksToPrune: %d, channelsToPrune: %d, clearState: %v, blocksToLoad: [%d, %d]}", s.blocksToPrune, s.channelsToPrune, s.clearState, s.blocksToLoad[0], s.blocksToLoad[1])
+		"SyncActions{blocksToPrune: %d, channelsToPrune: %d, clearState: %v, blocksToLoad: [%d, %d]}", s.blocksToPrune, s.channelsToPrune, s.clearState, s.blocksToLoad.start, s.blocksToLoad.end)
```

Comment:
This will panic if the range is `nil`.

Reply:
[b1f5fe7](https://github.com/ethereum-optimism/optimism/pull/13060/commits/b1f5fe7d6751871deca5da95c2f5d0d691689cca)